index.js

```diff
@@ -2,20 +2,37 @@ var domToReact = require('./lib/dom-to-react');
 var htmlToDOM = require('html-dom-parser');
 
 // decode HTML entities by default for `htmlparser2`
-var domParserOptions = { decodeEntities: true, lowerCaseAttributeNames: false };
+var defaultDomParserOptions = {
+  decodeEntities: true,
+  lowerCaseAttributeNames: false
+};
 
 /**
  * Convert HTML string to React elements.
  *
  * @param  {String}   html              - The HTML string.
  * @param  {Object}   [options]         - The additional options.
  * @param  {Function} [options.replace] - The replace method.
+ * @param  {Object}   [options.domParserOptions] - Additional domParserOptions options.
  * @return {ReactElement|Array}
  */
 function HTMLReactParser(html, options) {
   if (typeof html !== 'string') {
     throw new TypeError('First argument must be a string');
   }
+
+  // adding new options to domParserOptions
```

Comment:
**chore:** change line to `// add and/or update domParserOptions`

Reply:
I was still working on the code.